### PR TITLE
Update token.mdx with code examples

### PR DESCRIPTION
**Pull request by alionline**
Hello guys,

can you please review and let me know if there is something wrong/missing?

### CODE REVIEW

File: learn/code-examples/token/token.mdx [hunk 1]
```diff
@@ -8,65 +8,488 @@ date: "2021-12-01"
...
-};
+# TokenModule Code Examples
+
+Don’t forget to replace `<TOKEN_CONTRACT_ADDRESS>`, `<OWNER_ADDRESS>`, `<SPENDER_ADDRESS>` and `<SPENDER_ADDRESS_2>` with your own values. 
+
+### allowance(spender)
+
+```tsx
+// Set variable for the token module contract address which can be found after creating a token module in the dashboard
+const tokenAddress = "<TOKEN_CONTRACT_ADDRESS";
```
Comment (nachoiacovino) [inline learn/code-examples/token/token.mdx:19]:
```suggestion
const tokenAddress = "<TOKEN_MODULE_ADDRESS>";
```

this goes for everything "contract" -> "module"

**Review by nachoiacovino — CHANGES_REQUESTED** [review]
contracts -> modules

we should use async/await for everything from now on.

Also, can you pull from main and fix the conflicts?

File: learn/code-examples/token/token.mdx (hunk 1 above)
Comment (alionline) [reply learn/code-examples/token/token.mdx]: hey i processed the feedback, can you please check again ?

**Comment by nachoiacovino** [comment]
Fixed on commit https://github.com/nftlabs/portal/commit/472635536c826b32c1f8f6f9e68ece6b80f2a11d

**Pull request closed without merging**